### PR TITLE
Hw dao console handler

### PR DESCRIPTION
**Pull request by Likh-Alex**

### CODE REVIEW

**Review by Alex-Korzun — APPROVED** [review]

File: src/main/java/contoller/BetConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                    int betValue = Integer.parseInt(betData[BET_VALUE]);
+                    int betRisk = Integer.parseInt(betData[BET_RISK]);
+                    newBet = new Bet(betValue, betRisk);
+                } catch (ArrayIndexOutOfBoundsException e) {
+                    System.out.println("Please enter both bet and risk");
+                }
+            } catch (NumberFormatException nfe) {
+                System.out.println("Please enter valid value and risk");
+            }
+            betDao.add(newBet);
```
Comment (KseniiaMakarova) [inline src/main/java/contoller/BetConsoleHandler.java:34]: let's add bet to the storage if it is not null

File: src/main/java/dao/implementation/PersonDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package dao.implementation;
+
+import dao.PersonDao;
+import db.Storage;
+import java.util.List;
+import model.Person;
+
+public class PersonDaoImpl implements PersonDao {
+
```
Comment (KseniiaMakarova) [inline src/main/java/dao/implementation/PersonDaoImpl.java:9]: redundant empty line

File: src/main/java/model/Person.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package model;
+
+public class Person {
+    String name;
+    int age;
```
Comment (KseniiaMakarova) [inline src/main/java/model/Person.java:5]: what is missing?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/Main.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import contoller.BetConsoleHandler;
+import contoller.PersonConsoleHandler;
+
+public class Main {
+    public static void main(String[] args) {
+
```
[the remark this answers is not in the document]
Comment (Likh-Alex) [reply src/main/java/Main.java:6]: Done

File: src/main/java/dao/implementation/PersonDaoImpl.java (hunk 2 above)
Comment (Likh-Alex) [reply src/main/java/dao/implementation/PersonDaoImpl.java:9]: Done

File: src/main/java/model/Person.java (hunk 3 above)
Comment (Likh-Alex) [reply src/main/java/model/Person.java:5]: Access modifiers) Fixed

File: src/main/java/contoller/BetConsoleHandler.java (hunk 1 above)
Comment (Likh-Alex) [reply src/main/java/contoller/BetConsoleHandler.java:34]: Done

File: src/main/java/contoller/BetConsoleHandler.java [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    public void handleBet() {
+        System.out.println("Please enter your bet and risk");
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet newBet = null;
+            try {
+                try {
```
[the remark this answers is not in the document]
Comment (Likh-Alex) [reply src/main/java/contoller/BetConsoleHandler.java:23]: Done, refactored for two catch blocks

File: src/main/java/contoller/BetConsoleHandler.java [hunk 6]
```diff
@@ -0,0 +1,38 @@
+package contoller;
+
+import dao.BetDao;
+import dao.implementation.BetDaoImpl;
+import java.util.Scanner;
+import model.Bet;
+
+public class BetConsoleHandler {
```
[the remark this answers is not in the document]
Comment (Likh-Alex) [reply src/main/java/contoller/BetConsoleHandler.java:8]: Done, moved handling of the `Person` also into the `ConsoleHandler`, now there is only one `ConsoleHandler` with one `handle` method.

File: src/main/java/contoller/BetConsoleHandler.java [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    public static final int BET_RISK = 1;
+    public static final int BET_VALUE = 0;
+    BetDao betDao = new BetDaoImpl();
+    Scanner scanner = new Scanner(System.in);
```
[the remark this answers is not in the document]
Comment (Likh-Alex) [reply src/main/java/contoller/BetConsoleHandler.java:12]: Missing access modifiers. Made the `scanner` a local variable

File: src/main/java/dao/PersonDao.java [hunk 8]
```diff
@@ -0,0 +1,14 @@
+package dao;
+
+import java.util.List;
+import model.Bet;
+import model.Person;
+
+public interface PersonDao {
+
```
Comment (KseniiaMakarova) [inline src/main/java/dao/PersonDao.java:8]: redundant empty line

File: src/main/java/dao/PersonDao.java [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+    void add(Person person);
+
+    Bet getBet(Person person);
```
Comment (KseniiaMakarova) [inline src/main/java/dao/PersonDao.java:11]: let's remove this method for now and not bind Person and Bet

File: src/main/java/dao/implementation/PersonDaoImpl.java [hunk 10]
```diff
@@ -0,0 +1,26 @@
+package dao.implementation;
+
+import dao.PersonDao;
+import db.Storage;
+import java.util.List;
+import model.Bet;
+import model.Person;
+
+public class PersonDaoImpl implements PersonDao {
+
```
Comment (KseniiaMakarova) [inline src/main/java/dao/implementation/PersonDaoImpl.java:10]: redundant empty line - please remove

File: src/main/java/model/Person.java [hunk 11]
```diff
@@ -0,0 +1,44 @@
+package model;
+
+public class Person {
+    private String name;
+    private int age;
+    private Bet bet;
```
Comment (KseniiaMakarova) [inline src/main/java/model/Person.java:6]:
```suggestion
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/dao/PersonDao.java (hunk 8 above)
Comment (Likh-Alex) [reply src/main/java/dao/PersonDao.java:8]: Deleted

File: src/main/java/dao/PersonDao.java (hunk 9 above)
Comment (Likh-Alex) [reply src/main/java/dao/PersonDao.java:11]: Removed

File: src/main/java/dao/implementation/PersonDaoImpl.java (hunk 10 above)
Comment (Likh-Alex) [reply src/main/java/dao/implementation/PersonDaoImpl.java:10]: Removed as well, thank you.

File: src/main/java/model/Person.java (hunk 11 above)
Comment (Likh-Alex) [reply src/main/java/model/Person.java:6]: removed with methods as well.

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**